src/Screens/Main/Settings/UserAccount/MentorForm.tsx

```diff
@@ -67,6 +106,40 @@ export default ({ userId }: Props) => {
           account.vacation.switch"
         />
       )}
+      <Message
+        style={styles.fieldName}
+        id="main.settings.account.status.title"
+        testID="main.settings.account.status.title"
+      />
+      {pipe(
+        requestState,
+        RD.fold(
+          () => (
+            <StatusMessageForm
+              statusMessage={statusMessage}
+              setStatusMessage={setStatusMessage}
+              onButtonPress={changeStatusMessage}
+            />
+          ),
+          () => <Spinner />,
+          () => (
+            <AlertBox
+              imageStyle={styles.failBox}
+              imageSource={require('../../../images/alert-circle.svg')}
+              duration={changeStatusMessageState.coolDownDuration}
+              messageId="main.settings.account.status.fail"
+            />
```

Comment:
We could modify this AlertBox component, or then create a brand new one. There could take two functions, one for retrying the request and one for resetting the state =)

(IF we want to have it that way that the state is not resetted automatically after a timeout)